### PR TITLE
Adding output value

### PR DESCRIPTION
**Pull request by eBsowka**
## Overview

Adds job id as an output from the action. 

## Changes

- Adds new parameter in: `action.yaml`
- Adds output in: `entrypoint.sh`

## How to test

New parameter can be seen in: (in the testing section)
run: https://github.com/lifebit-ai/etl_profile_clean/actions/runs/3157966904/jobs/5139457317
code: https://github.com/lifebit-ai/etl_profile_clean/blob/97d5e617df32289cd9021e32b648b0dbc376379d/.github/workflows/cloudos.yml#L36

## PR Checklist:

I confirm that:

- [ ] all changes/additions are documented
- [ ] ci tests added

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -71,6 +71,11 @@ inputs:
     description: 'Mode of execution for the action, by default the API call will be sent. Set to dry_run: true if you only want to print the command (strips secrets before printing).'
     required: false
     default: ''
+outputs:
+  jobid:
```
Comment (cgpu) [inline action.yaml:75]:
```Suggestion
  job-id
```

Comment (eBsowka) [reply action.yaml:75]: 👍

**Review by mmeier93 — APPROVED** [review]
Approving 👍

**Comment by cgpu** [comment]
@eBsowka @mmeier93 
This PR shouldn't have been merged because
- The branch was not checked out from the latest up to date branch
- It does not document the change that was introduced
- The CI test is only doing a dry run, which is not sufficient to test the functionality. In a respective PR that swicthes `dry_run` to false the test fails, see here: https://github.com/lifebit-ai/action-cloudos-cli/actions/runs/3338208559/jobs/5525572591

For this reason we are going to peel off this squash commit from `dev` and remake the changes in a newly created branch with the latest changes.

**Pull request merged**